config.json

```diff
@@ -1869,6 +1869,14 @@
           "lists"
         ],
         "difficulty": 10
+      },
+      {
+        "slug": "paasio",
+        "name": "PaaS I/O",
+        "uuid": "81177978-2ed3-47c2-a6a0-fef316d94c0b",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

Comment:
This really shouldn't be a one (it's not that easy). Perhaps we could use the Go track as a guide - they have used 4 as the difficulty.